public/apps/configuration/panels/permission-list/permission-list.tsx

```diff
@@ -67,21 +69,17 @@ function renderBooleanToCheckMark(value: boolean): React.ReactNode {
 }
 
 function toggleRowDetails(
-  item: ActionGroupListingItem,
+  item: PermissionListingItem,
+  actionGroupDict: DataObject<ActionGroupItem>,
```

Comment:
Minor: curious what "Dict" stands for. Probably just call it actionGroupItems or actionGroups.

Reply:
I add suffix dict to indicate it is a dict for building tree.